go.mod

```diff
@@ -9,23 +9,22 @@ require (
 	github.com/cpuguy83/go-md2man v1.0.10
 	github.com/docker/cli v20.10.8+incompatible
 	github.com/docker/docker v20.10.8+incompatible
-	github.com/fatih/color v1.9.0
+	github.com/fatih/color v1.13.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/google/go-cmp v0.5.6
-	github.com/google/go-containerregistry v0.5.2-0.20210709161016-b448abac9a70
+	github.com/google/go-containerregistry v0.6.0
 	github.com/hako/durafmt v0.0.0-20191009132224-3f39dc1ed9f4
 	github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c
-	github.com/jonboulle/clockwork v0.1.1-0.20190114141812-62fb9bc030d1
+	github.com/jonboulle/clockwork v0.2.2
 	github.com/ktr0731/go-fuzzyfinder v0.2.0
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/opencontainers/image-spec v1.0.2 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
-	github.com/tektoncd/hub/api v0.0.0-20210922050526-aa589697cb58
-	github.com/tektoncd/pipeline v0.28.0
-	github.com/tektoncd/plumbing v0.0.0-20210514044347-f8a9689d5bd5
-	github.com/tektoncd/triggers v0.16.0
+	github.com/tektoncd/hub/api v0.0.0-20210315114749-a667b06c02ac
```

Comment:
@pradeepitm12 can you please bump the latest version of hub? This commit belongs to 5th March, 2021

Reply:
copy paste from other pr. :stuck_out_tongue: